docs/cli/profile.md

```diff
@@ -0,0 +1,31 @@
+## ⏱️ Profiling Models
+
+`nos profile` supports model benchmarking across a number of axes including iterations per second,
+memory footprint and utilization on CPU/GPU.
+Currently, the nos profiler itself runs natively in the execution environment (i.e. outside
+the NOS server), so you'll need to install both the `server` and `test` dependencies alongside
+your existing NOS installation 
+
+```bash
+pip install torch-nos[server,test]
+```
+
+If you have the time, you can construct a profiling catalog on your machine in its entirety with:
+```bash
+nos profile rebuild-catalog
+```
+
+You can also profile specific models with 
+```bash
+nos profile method -m openai/clip-vit-large-patch14
+```
+
+Or an entire method type with
+```bash
+nos profile method -m encode_image
```

Comment:
Are method type and method names interchangeable here with `-m` flag? Wouldn't we want them to have a different flag? `-m` for model name and `-t` for method type?